static/js/page_params.ts

```diff
@@ -16,6 +16,7 @@ export const page_params: {
     realm_add_custom_emoji_policy: number;
     realm_avatar_changes_disabled: boolean;
     realm_create_stream_policy: number;
+    realm_delete_own_message_policy: number;
```

Comment:
Do we also want to add types for all the fields in `page_params` here as we did for `user_settings`?

Reply:
Yes, we do; I believe `page_params` is incomplete only because @priyank-p hadn't had time to do a complete audit and include everything there, and thus was only adding what was needed when integrating new modules.  It'd be a great little project to make `page_params` complete.